android/src/test/java/com/revenuecat/purchases/common/PeriodTests.kt

```diff
@@ -0,0 +1,24 @@
+package com.revenuecat.purchases.common
+
+import android.os.Build
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.annotation.Config
+
+@RunWith(AndroidJUnit4::class)
+@Config(sdk = [Build.VERSION_CODES.P])
+class PeriodTests {
```

Comment:
Couldn't use `Spek` because it's not compatible with `Roboelectric`. Or at least I couldn't get it to work https://github.com/spekframework/spek/issues/207

Reply:
so essentially anything that logs is just not compatible with Spek? if so, we might wanna stop using it altogether

Reply:
not for this pr though

Reply:
I will give another try another day to try to run RoboElectric + Spek. Either way, I am still not convinced about using Spek, I had another problem couple weeks back because their IDE plugin wasn't updated and was giving me issues with regular unit tests, and I had to run all tests from the command line. Anyhow, that's a discussion for another day

Reply:
Alternative to adding RoboElectric would be to mock the Log class, which we can also do. I can try that too